src/Sparrow.Server/LowMemory/CheckPageFileOnHdd.cs

```diff
@@ -421,6 +421,8 @@ public static string PosixIsSwappingOnHddInsteadOfSsd()
                     HashSet<string> disks = KernelVirtualFileSystemUtils.GetAllDisksFromPartitionsFile();
                     foreach (var disk in disks)
                     {
+                        //ignore ramdisks (ram0..ram15 etc)
+                        if (disk.Equals("ram", StringComparison.OrdinalIgnoreCase)) continue;
```

Comment:
Needs disk.StartWith instread of equals.
Also start with "loop" (loop0-19) and start with "sr" (cdrom). They are not failing this but they needed to be ignored